js/src/styles/ant-vars.less

```diff
@@ -1,162 +1,162 @@
-@import 'antd/es/style/themes/default.less';
-
 @dark-gray: #313131;
 @purple: #8048f3;
 @light-gray: #6e6e6e;
 @bright-green: #b2d030;
 
 // -------- Colors -----------
-@primary-color : tint(@purple, 90%);
-@success-color : @bright-green;
-@highlight-color : @purple;
-@warning-color : @gold-6;
-@normal-color : @light-gray;
-@background-color : @dark-gray;
-@disabled-color : fade(@light-gray, 10%);
-@disabled-bg : @background-color;
+@primary-color: tint(@purple, 90%);
+@success-color: @bright-green;
+@highlight-color: @purple;
+@warning-color: @gold-6;
+@normal-color: @light-gray;
+@background-color: @dark-gray;
```

Comment:
Are all the lines in this file actually needed?  Is it much different than the built-in styling? Just wondering if we could even remove most of this file.

Reply:
Seems likely that we could trim a lot of it down, though in that case we'd be looking at resolving this across multiple repositories. (The diffs here are just from copying the version of this file in web-3d-viewer over this one.)

Really web-3d-viewer ought to be responsible for these variables, it makes no sense that users of this component would have to jump through these hoops. The fact that they do have to is in part down to how the viewer kind of fights ant's styling rather than playing nice with it (an issue which I've unfortunately perpetuated a bit). I've gone back and forth on how best to clean this up, and I'm currently thinking the best solution long-term is probably to move the viewer's styling to less so that it can use ant's variables, then provide a default theme file that looks like this and can be imported out of the viewer package.

Reply:
makes sense.  sounds like this file is good to go until we do the bigger change.